### DISCUSSION TITLE
Lab Fee Schedule for "Lab Only" Outpatient Claims

### DISCUSSION THREAD

**Discussion by foustsWCH** [CMSgov/hospital-price-transparency#45] [Q&A] [answered]
We have a contract that has a fixed lab fee schedule that only applies to outpatients claims that are "labs only."  If, on the other hand, the patient has an outpatient encounter and receives other services as well, the fee schedule does not apply, and reimbursement is based on whatever methodology is used for other outpatient services.

I want to make sure I understand the appropriate way to display this in a CSV wide format.  I have tried to review the examples and guidance already provided.  The closest scenario appears to be the revenue code 762 example where the reimbursement varies based on "rule out myocardial infarction" vs "without rule out myocardial infarction", as clarified in the "Additional payer notes."

If following that example is an appropriate model in this circumstance as well, is an acceptable way to display this data to create two rows for each lab and specifying the circumstances in the additional payer notes (i.e. "lab only encounter" vs "encounter with other services")?

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/fousts/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

code |   | negotiated_dollar | negotiated_percentage | negotiated_algorithm | estimated_amount | methodology | additional_payer_notes
-- | -- | -- | -- | -- | -- | -- | --
8000X | CPT | 4 |   |   |   | fee schedule | When   outpatient encounter is only for labs
8000X | CPT | 87 |   |   |   | percent of charge | When   outpatient encounter includes other services in addition to labs

</body>

</html>

**Comment by RidenourK** [comment, marked as answer]
@foustsWCH. Thank you for your question. 

CMS is unable to comment on or provide individualized/case-by-case predeterminations as to whether a particular strategy or approach would be deemed compliant. The compliance process typically involves a comprehensive compliance review in response to a CMS audit or a complaint received through the Hospital Price Transparency website.

However, CMS has provided examples of how hospitals could add contextual information to their MRFs explaining different standard charges for the same code(s) within a payer and plan. Please refer to rows 12-13 in the CSV Wide examples, rows 14-16 in the CSV Tall examples, and lines 174-185 in the JSON examples.

**Comment by foustsWCH** [comment]
Thank you for referring me to rows 12-13 of the CSV Wide examples, which appear to be what I had referenced above (Revenue Code 762).  I understand you are not officially allowed to deem any specific approach compliant outside of an official audit, but I appreciate the confirmation that the examples I had referenced would be a helpful model for reporting scenarios where additional contextual information determines the negotiated payment.
